### PR TITLE
chore(ilp): Handle trying to convert ILP strings into non STRING colu…

### PR DESCRIPTION
**Pull request by patrickSpaceSurfer**
…mns more gracefully

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by patrickSpaceSurfer** [comment]
> #### [PR Coverage check]
> 
> heart_eyes **pass** : 0 / 0 (0%)

**Comment by patrickSpaceSurfer** [comment]
> #### [PR Coverage check]
> 
> heart_eyes **pass** : 0 / 0 (0%)

I have no idea what this means?

**Comment by ideoma** [comment]
#### [PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java [hunk 1]
```diff
@@ -695,8 +719,16 @@ void processMeasurementEvent(WriterJob job) {
...
                             job.floatingCharSink.asCharSequence(bufPos, hi);
-                            row.putStr(colIndex, job.floatingCharSink);
                             bufPos = hi;
+                            final int colType = writer.getMetadata().getColumnType(colIndex);
+                            switch (colType) {
+                                case ColumnType.STRING:
+                                    row.putStr(colIndex, job.floatingCharSink);
+                                    break;
+                                default:
+                                    throw CairoException.instance(0).put("line protocol STRING cannot be inserted into column type ").put(ColumnType.nameOf(colType)).put(") [entityType=")
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java:729]: logging `colType` twice

File: core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java [hunk 2]
```diff
@@ -695,8 +719,16 @@ void processMeasurementEvent(WriterJob job) {
                             bufPos += Integer.BYTES;
                             long hi = bufPos + 2L * len;
                             job.floatingCharSink.asCharSequence(bufPos, hi);
-                            row.putStr(colIndex, job.floatingCharSink);
                             bufPos = hi;
+                            final int colType = writer.getMetadata().getColumnType(colIndex);
+                            switch (colType) {
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java:724]: if - else perhaps?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
small changes pls

**Comment by ideoma** [comment]
#### [PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by ideoma** [comment]
#### [PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by bluestreak01 — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by bluestreak01 — APPROVED** [review]
Approve. Clean test-only fix — unique dummy-table names per test eliminate the log-barrier race on the reused server.

**Pull request merged**